exercises/palindrome-products/canonical-data.json

```diff
@@ -155,8 +157,8 @@
               "max": 1
             },
             "expected": {
-                "error": "invalid input: min is 2 and max is 1"
+                "error": "min must be <= max"
             }
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
Very minor nit pick.  This change (probably unintentional) is not related and should not be included.

Reply:
Ok, it is removed